hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/virtual/UniqueTokenValue.java

```diff
@@ -113,11 +114,10 @@ public static UniqueTokenValue from(final MerkleUniqueToken token) {
     }
 
     /**
-     * Returns the size in bytes of the class (if fixed) or {@link
-     * DataFileCommon#VARIABLE_DATA_SIZE} if variable sized.
+     * Returns the size in bytes of the class (if fixed) or {@link BaseSerializer#VARIABLE_DATA_SIZE} if variable sized.
      */
     public static int sizeInBytes() {
```

Comment:
Usually, methods like this are defined in `VirtualKey` or `VirtualValue` subclasses only if they are of fixed size. For variable sized objects, `VARIABLE_DATA_SIZE` is returned from the corresponding `KeySerializer` or `ValueSerializer` classes. So this method looks redundant to me...

Reply:
I removed the method.